### PR TITLE
Deps update

### PR DESCRIPTION
**Pull request by Meklo**
## Major updates
They are listed in package.json diffs

## Minor updates and patches

Here's the list of dependencies whom minor updates and patches were applied :

@emotion/react	11.10.4	->	11.10.6
@emotion/styled	11.10.4	->	11.10.6
@mui/icons-material	5.10.3	->	5.11.16
@mui/lab	5.0.0-alpha.97	->	5.0.0-alpha.127
@mui/material	5.10.3	->	5.12.1
@mui/styles	5.10.3	->	5.11.16
@reduxjs/toolkit	1.8.5	->	1.9.5
ag-grid-community	29.0.0	->	29.2.0
ag-grid-react	29.0.0	->	29.2.0
core-js	3.25.0	->	3.30.1
eslint-config-prettier	8.5.0	->	8.8.0
plotly.js-basic-dist-min	2.18.2	->	2.20.0
prettier	2.7.1	->	2.8.7
qs	6.11.0	->	6.11.1
react-flow-renderer	10.3.16	->	10.3.17
react-hook-form	7.41.0	->	7.43.9
react-intl	6.1.0	->	6.4.1
react-redux	8.0.2	->	8.0.5
react-resizable	3.0.4	->	3.0.5
react-router-dom	6.3.0	->	6.10.0
react-window	1.8.7	->	1.8.9 
redux	4.2.0	->	4.2.1

webpack-dev-server is staying to the version 4.10.1 because the version 4.13.3 was causing error overlay issues

### CODE REVIEW

File: src/components/network-map-tab.js [hunk 1]
```diff
@@ -848,15 +848,17 @@ export const NetworkMapTab = ({
...
-            {displayOverloadTable && isLoadFlowValid() && linesNearOverload() && (
-                <div className={classes.divOverloadedLineView}>
-                    <OverloadedLinesView
-                        lineFlowAlertThreshold={lineFlowAlertThreshold}
-                        mapEquipments={mapEquipments}
-                        disabled={disabled}
-                    />
-                </div>
-            )}
+            {displayOverloadTable &&
```
Comment (sBouzols) [inline src/components/network-map-tab.js:851]: Something strange here ? whitespaces ?

File: src/components/dialogs/inputs/input-hooks.js [hunk 2]
```diff
@@ -191,11 +191,13 @@ export const useTextValue = ({
                 InputProps={{
                     endAdornment: (
                         <InputAdornment position="end">
-                            {clearable && value !== undefined && value !== '' && (
-                                <IconButton onClick={handleClearValue}>
-                                    <ClearIcon />
-                                </IconButton>
-                            )}
+                            {clearable &&
```
Comment (sBouzols) [inline src/components/dialogs/inputs/input-hooks.js:194]: same strange diff ? indentation change ? why

**Review by sBouzols — APPROVED** [review]

**Review by jonenst — APPROVED** [review]

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]
Code review: OK
Tests : 
Pb when doing a modification (opening/closing a switch for example)
Error screen is displayed with the following message : 

Uncaught Error: [@formatjs/intl] An `id` must be provided to format a message. You can either:
1. Configure your build toolchain with [babel-plugin-formatjs](https://formatjs.io/docs/tooling/babel-plugin)
or [@formatjs/ts-transformer](https://formatjs.io/docs/tooling/ts-transformer) OR
2. Configure your `eslint` config to include [eslint-plugin-formatjs](https://formatjs.io/docs/tooling/linter#enforce-id)
to autofix this issue
    invariant utils.js:76
    formatMessage message.js:31
    formatMessage provider.js:46
    FormattedMessage message.js:21
    React 13

File: src/components/network-map-tab.js (hunk 1 above)
Comment (Meklo) [reply src/components/network-map-tab.js:851]: It's linked to a prettier update I reckon

File: src/components/dialogs/inputs/input-hooks.js (hunk 2 above)
Comment (Meklo) [reply src/components/dialogs/inputs/input-hooks.js:194]: Same here

**Pull request merged**